apps/pie-docs/src/foundations/iconography/code.md

```diff
@@ -0,0 +1,277 @@
+
+```html
+<script src="path/to/dist/pie-icons.js"></script>
+```
+```html
+<!-- choose one -->
+<script src="https://unpkg.com/@justeattakeaway/pie-icons"></script>
+<script src="https://cdn.jsdelivr.net/npm/@justeattakeaway/pie-icons/dist/pie-icons.min.js"></script>
+```
+
+#### HTML
+Simply add a `data-pie-icons` attribute with the icon name to an element. You can see all of our icons on the [library page](http://localhost:8080/foundations/iconography/library/).
+
+```html
+<i data-pie-icons="menu"></i>
+```
+
+#### Invoke the package
+Calling the `replace()` function will replace all elements with the `data-pie-icons` attribute with the relevant icon.
+
+```html
+<script>
+  window['pie-icons'].default.replace();
+</script>
+```
+---
+
+### Usage - NodeJS
+
+#### Installation
+You can install the package using `npm` or `yarn`:
+
+```sh
+npm install @justeattakeaway/pie-icons --save
+```
+```sh
+yarn add @justeattakeaway/pie-icons
+```
+
+#### Require the package
+Simply require the package in your application where needed:
+
+```js
+const pieicons = require('@justeattakeaway/pie-icons');
+```
+
+#### Call the APIs
+The package exposes a number of APIs for you to use:
+
+```js
+pieicons.icons.x
+// {
+//    name: 'x',
+//    contents: '<line ... /><line ... />`,
+//    tags: ['cancel', 'close', 'delete', 'remove'],
+//    attrs: {
+//      class: 'c-pieIcon c-pieIcon--x',
+//      xmlns: 'http://www.w3.org/2000/svg',
+//    },
+//    toSvg: [Function],
+// }
+
+pieicons.icons.x.toSvg()
+// <svg class="c-pieIcon c-pieIcon--x" ...><line ... /><line ... /></svg>
+
+pieicons.icons.x.toSvg({ class: 'foo bar', 'stroke-width': 1, color: 'red' })
+// <svg class="c-pieIcon c-pieIcon--x foo bar" stroke-width="1" color="red" ...><line ... /><line ... /></svg>
+```
+
+{% notification {
+  type: "information",
+  message: "For more information about the APIs in this package, please take a look at the [API reference](https://github.com/justeattakeaway/pie/tree/main/packages/tools/pie-icons#api-reference) in the project README."
+} %}
+
+
+---
+
+## React
+
+This package generates an icon set for React applications using the base [pie-icons](https://www.npmjs.com/package/@justeattakeaway/pie-icons) package. The SVGs in pie-icons are compiled into React components.
+
+The icons are bundled in CommonJS and ES Modules, to be more easily adopted for modern React Applications.
+
+
+### Usage
+
+#### Installation
+You can install the package using `npm` or `yarn`:
+
+```sh
+npm install @justeattakeaway/pie-icons-react --save
+```
+```sh
+yarn add @justeattakeaway/pie-icons-react
+```
+
+#### Import into your project
+Simply import the icons you need into your project:
+
+```tsx
+import { IconAlertTriangleLarge, IconCalendar } from "@justeattakeaway/pie-icons-react";
+
+export default function App() {
+  return (
+    <div className="App">
+      <IconCalendar />
+      <IconAlertTriangleLarge fill={PIE_ALIAS_COLOR_TOKEN} />
+    </div>
+  );
+}
+```
+
```

Comment:
I would probably mention the `size` prop – likely at the end of all the packages (as I think it's the same across all of them) – but it could be linked here once it's added too